examples/openapi-spec-writer/src/mastra/tools/index.ts

```diff
@@ -110,7 +110,7 @@ export const generateSpecTool = createTool({
       throw new Error("No crawled data found");
     }
 
-    const agent = mastra?.getAgent("openapi-spec-gen-agent");
+    const agent = mastra?.agents?.["openapi-spec-gen-agent"];
```

Comment:
why dont we want to use the getter here

Reply:
there's no getter on `mastra`.
`mastra` is no longer a `Mastra` instance, it's now an simple object containing handpicked primitives

Reply:
alright then